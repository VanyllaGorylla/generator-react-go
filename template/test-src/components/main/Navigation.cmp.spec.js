import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

import Navigation from '@/components/main/Navigation.cmp';

import renderer from 'react-test-renderer';
describe(`Navigation.cmp test`, function() {
  it(`should proper HTML`, function() {
    const tree = renderer
      .create(
        <Router>
          <Navigation />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
