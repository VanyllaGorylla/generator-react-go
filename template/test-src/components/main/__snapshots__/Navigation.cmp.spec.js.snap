// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation.cmp test should proper HTML 1`] = `
<nav>
  <div
    className="nav-wrapper"
  >
    <a
      className="brand-logo "
      href="/"
    >
      apka
    </a>
    <ul
      className="right hide-on-med-and-down"
    >
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          Index
        </a>
      </li>
      <li>
        <a
          href="/todo-example"
          onClick={[Function]}
        >
          Todo Example
        </a>
      </li>
      <li>
        <a
          href="/posts"
          onClick={[Function]}
        >
          Posts
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
